test/unit/test-utils.js

```diff
@@ -0,0 +1,31 @@
+const _ = require('lodash');
```

Comment:
How about merging this file with `test/unit/zendesk-uploader/test-utils.js` and move to same hierarchy level with `unit` and `functional` tests folders? This will make all data stubs + utilities to be gathered in one place. Or this is a bad idea to mix it?